packages/keystone/lib/Keystone/index.js

```diff
@@ -157,10 +157,15 @@ module.exports = class Keystone {
       // We do it within the request callback so we can resolve it based on the
       // request info ( like who's logged in right now, etc)
       getCustomAccessControlForUser = memoize(
-        async access => {
+        async (item, args, context, info, access, gqlName) => {
```

Comment:
Is this a major breaking change? Is `getCustomAccessControlForUser` considered part of our external API?

Reply:
Gosh I hope not. If someone's digging that deep then... 🤷 I think it's safe to call this an internal function, even if it is technically publicly available.